pulsar/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display_name": "pulsar",
+  "maintainer": "ming.luo@kesque.com",
+  "manifest_version": "1.0.0",
+  "name": "pulsar",
+  "metric_prefix": "pulsar.",
```

Comment:
```suggestion
  "metric_prefix": "kesque.pulsar.",
```

Reply:
@ofek This is for a generic Open Source Apache Pulsar's metric. Is the company name `kesque` still required?

Reply:
Yup! That's our new standard

Reply:
Thanks! Added